open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservices/assetcatalog/admin/AssetCatalogServicesInstance.java

```diff
@@ -34,7 +34,8 @@
     AssetCatalogServicesInstance(OMRSRepositoryConnector repositoryConnector,
                                  List<String> supportedZones, OMRSAuditLog auditLog,
                                  String localServerUserId, List<String> supportedTypesForSearch) throws org.odpi.openmetadata.commonservices.multitenant.ffdc.exceptions.NewInstanceException {
-        super(description.getAccessServiceName(), repositoryConnector, auditLog, localServerUserId, repositoryConnector.getMaxPageSize());
+
+        super(description.getAccessServiceName() + " OMAS ", repositoryConnector, auditLog, localServerUserId, repositoryConnector.getMaxPageSize());
```

Comment:
Looks like there is an extra space in the string of` " OMAS "`. And also curious why we need to hard code it on OMASes?